File: PASCAL/Micro08.pas
program micro08 ;
var numero : Integer;
begin
    numero := 1 ;
    while( numero <> 0 ) do
    begin
        write( ' Digite um numero : ' ) ;
        readln( numero ) ;
        if( numero > 10 ) then
        begin
            write( 'O numero , ' ) ;
            write( numero ) ;
            writeln( ' e maior que 10. ' ) ;
            end
            else
            begin
                write( 'O numero , ' ) ;
                write( numero ) ;
                writeln( ' e menor que 10' ) ;
        end ;
    end ;
end .

File: IR LLVM/Micro08/Micro08.c
#include <stdio.h>
#include <stdlib.h>


int main(){
    int num = 1;

    while(num != 0){
        printf("Digite um numero: ");
        scanf("%d", &num);
        if(num > 10)
            printf("O numero %d é maior que 10\n", num);
        if(num < 10)
            printf("O numero %d é menor que 10\n", num);
    }
    return 0;
}